_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
+Then we can push them to our forked version of the repo.
 
 ~~~
 $ git add france.txt
 $ git commit -m "Added largest city to france file"
-$ git push origin master
+$ git push origin addFrance
 ~~~
 {: .bash}
 
+~~~
+Counting objects: 3, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (3/3), 387 bytes | 0 bytes/s, done.
+Total 3 (delta 0), reused 0 (delta 0)
+To https://github.com/sstevens2/countries.git
+   31aa2e3..609acfe  addFrance -> addFrance
+~~~
+{: .output}
+
 Now if we reload the pull request.  The new commit was added to that pull request.
+New commits pushed to the same branch are included in the previous pull request.
+If you want to suggest changes separately you need to make separate branches but 
+if you want the changes to be considered together you should put them in the same branch.
+
+![](../fig/github_screenshot_after_new_commit.png)
+
+When working with others we might encounter the conflicts, which we
+learned about earlier in branches.  Let's practice resolving conflicts when working
+collaboratively.
+
+We will continue to work in the `addFrance` branch from before and check we are
+in that branch before we start.
+
+
+~~~
+$ git branch
+~~~
+{: .bash}
+
+~~~
+* addFrance
+  master
+~~~
+{: .output}
+
+
+Next we will each add our country to the existing `README.md` file in the repository in the line directly following the `Countries:` line.
```

Comment:
I really like the concept of creating a conflict in the README. It's a great way to introduce README's in repos and their automatic rendering on GH. (This may be a repeat comment. If so, still love it. :smile: )